src/index.js

```diff
@@ -92,18 +93,20 @@ class Index extends Component {
                             <Route path={`${rootPath}/collections/:collectionID/preview`} component={ userIsAuthenticated(PreviewController) }/>
 
                             {config.enableDatasetImport === true &&
-                                <Route path={`${rootPath}/collections/:collectionID/datasets`} >
-                                    <IndexRoute component={ userIsAuthenticated(SelectADataset) }/>
-                                    <Route path=':datasetID'>
-                                        <IndexRoute component={ userIsAuthenticated(DatasetEditionsController) }/>
-                                        <Route path={`editions`} component={ userIsAuthenticated(CreateEditionController) }/>
-                                        <Route path='editions/:editionID'>
-                                            <Route path={`instances`} component={ userIsAuthenticated(CreateVersionController) }/>
-                                            <IndexRoute component={ userIsAuthenticated(DatasetVersionsController) }/>
-                                            <Route path={`versions/:versionID`} component={ userIsAuthenticated(DatasetMetadataController) }>
-                                                <Route path={`edit/:metadataField/:metadataItemID`} component={ userIsAuthenticated(EditMetadatItem) }/>
+                                <Route component={ CollectionRoutesWrapper }>
+                                    <Route path={`${rootPath}/collections/:collectionID/datasets`} >
+                                        <IndexRoute component={ userIsAuthenticated(SelectADataset) }/>
+                                        <Route path=':datasetID'>
+                                            <IndexRoute component={ userIsAuthenticated(DatasetEditionsController) }/>
+                                            <Route path={`editions`} component={ userIsAuthenticated(CreateEditionController) }/>
+                                            <Route path='editions/:editionID'>
+                                                <Route path={`instances`} component={ userIsAuthenticated(CreateVersionController) }/>
+                                                <IndexRoute component={ userIsAuthenticated(DatasetVersionsController) }/>
+                                                <Route path={`versions/:versionID`} component={ userIsAuthenticated(DatasetMetadataController) }>
+                                                    <Route path={`edit/:metadataField/:metadataItemID`} component={ userIsAuthenticated(EditMetadatItem) }/>
```

Comment:
Typo: EditMetadatItem -> EditMetadataItem (initial type wasn't on this PR but just noticed it, also on the import itself)

There is also another in this file just noticed but is out of the scope of PR (feel free to update it though :P 'userisAdminOrEditor' -> 'userIsAdminOrEditor'

Reply:
Fixed